docs/events-interception.md

```diff
@@ -0,0 +1,190 @@
+# About events interception {#events-interception}
+
+Testplane allows the developer to intercept a number of events and change them to others, ignore or delay their processing.
+
+Events that can be intercepted are provided with the _can be intercepted_ tag in the description. There are 7 such events:
```

Comment:
*interceptable* is a shorter alternative. Not critical

Reply:
OK, agree